### ISSUE TITLE
Not sure what banko is?

### ISSUE THREAD

**Issue by JonahAragon** [bankogang/main-web#1]
hello sirs? i name is jonah and i do not know what this banko is or what it does or what it is doing here on instagram plz explain this to me sirs please and thank you. 

- mr jonahs iranian twin

**Comment by k-c-d** [comment]
do you have cancer? please, leave me alone.

**Comment by JonahAragon** [comment]
sirs i do not hear a reply from you guys plx respond

**Comment by JonahAragon** [comment]
i do have cancer but it is only in zodiac sirs

**Comment by k-c-d** [comment]
let's just give each other chemo and move on

**Comment by JonahAragon** [comment]
staring chemo to you sir do you feel chemo?

**Comment by k-c-d** [comment]
OH MY GOD THIS IS SO CANCEROUT

**Comment by JonahAragon** [comment]
yes?

**Comment by k-c-d** [comment]
do u wanna fuck?

**Comment by JonahAragon** [comment]
fam please i have family and 3 kids no sir here

**Comment by k-c-d** [comment]
wanna see my tumor??

**Comment by k-c-d** [comment]
no u cant volunteer me like that

**Comment by JonahAragon** [comment]
If you want to see my lung!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!11111111 I keep it in a jar

**Comment by k-c-d** [comment]
# CANCER NIHAR HAS CANCER

**Comment by k-c-d** [comment]
# GUESS WHAT NIHAR?

**Comment by JonahAragon** [comment]
# *WHY ARE WE SCREAMING?*

**Comment by k-c-d** [comment]
# FUCK YOU _AND_ YOUR FUCKING TUMOR

**Comment by JonahAragon** [comment]
# THIS IS TOO MANIC WHY IS THIS CONTINUING?

**Comment by JonahAragon** [comment]
# I REGRET EVERYTHING

**Comment by k-c-d** [comment]
# DAWG IF I WAS U, I'D KILL MYSELF

**Comment by JonahAragon** [comment]
# LAST WORD.

**Comment by k-c-d** [comment]
# MADE A MIL MYSELF, AND I'm STILL MYSELF

**Comment by k-c-d** [comment]
# AND I'LL LOOK IN THE MIRROR IF I NEED SOME HELP

**Comment by JonahAragon** [comment]
# STOP IS LOCKED

**Comment by k-c-d** [comment]
# KILL YOURSELF DAWG

**Comment by k-c-d** [comment]
# forgive english, i am Russia.
i come to study clothing and fashion at American university. i am here little time and i am very hard stress. i am gay also and this very difficult for me, i am very religion person. i never act to be gay with other men before. but after i am in america 6 weeks i am my friend together he is gay also. He was show me American fashion and then we are kiss.
We sex together. I never before now am tell my mother about gay because i am very shame. As i fock this American boy it is very good to me but also i am feel so guilty. I feel extreme guilty as I begin orgasm. I feel so guilty that I pick up my telephone and call Mother in Russia. I awaken her. It too late for stopping so I am cumming sex. I am very upset and guilty and crying, so I yell her, "I AM CUM FROM SEX" (in Russia). She say what? I say "I AM CUM FROM SEX" and she say you boy, do not marry American girl, and I say "NO I AM CUM FROM SEX WITH MAN, I AM IN ASS, I CUM IN ASS" and my mother very angry me. She not get scared though.
I hang up phone and am very embarrass. My friend also he is very embarrass. I am guilt and feel very stupid. I wonder, why do I gay with man? But I continue because when it spurt it feel very good in American ass.

**Comment by k-c-d** [comment]
I don't believe in judgement of people based on their personal choices as long as those personal choices aren't harming others. I don't have any issue with any type of sexuality whatsoever (including physically acting out necrophilia, pedophilia, or other acts which have a harmful affect on others - but I don't care what a person's fantasies consist of, as long as they recognize the difference between reality and fiction and can separate them). I don't have any issue with anybody over what type of music they listen to, or clothes they wear, etc. I know that's not really an impressive moral, but it's unfortunately rare; a great many people, especially those my age, are judgmental about these things.

**Comment by k-c-d** [comment]
i am genius

**Comment by k-c-d** [comment]
wanna fuck

**Comment by k-c-d** [comment]
no judgement let's all be pedophiles!

**Comment by k-c-d** [comment]
i am pedophile @GitHub

**Comment by k-c-d** [comment]
close down mi accounto, i dare u
